app/components/show/item/details_component.html.erb

```diff
@@ -39,7 +39,7 @@
       <tr>
         <th class="col-3" scope="row">Released to</th>
         <td>
-            <%= released_to %>
+            <%= user_version_view? ? 'Older versions are not released' : released_to %>
```

Comment:
Will this message also show on the current version?

Reply:
Yes. I think this is OK for now, as @andrewjbtw is going to be reconsidering the display of non-version specific fields like "Status".